homeassistant/components/sql/config_flow.py

```diff
@@ -125,6 +125,55 @@ async def async_step_user(
                     },
                 )
 
+            return self.async_show_form(
```

Comment:
To avoid the duplicate code you can make a function that will construct the `vol.Schema` object and take a `dict` of suggested values

Reply:
Please see if it's better...

If it is not in the expected pattern, can you indicate an integration that implements data schema manipulation so that I can follow the same pattern?